File: AppMovil/app/src/main/java/com/example/appmovil/model/Empleado.java
package com.example.appmovil.model;

public class Empleado {

    private int idEmpleado;
    private String nombre;
    private String apellido;

    public void setIdEmpleado(int idEmpleado) {
        this.idEmpleado = idEmpleado;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public int getIdEmpleado() {
        return idEmpleado;
    }

    public String getNombre() {
        return nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public Empleado(int idEmpleado, String nombre, String apellido)
    {

        this.idEmpleado = idEmpleado;
        this.nombre = nombre;
        this.apellido = apellido;
    }

    public Empleado()
    {
    }
}

File: BackendPostgres/BackendPostgres/Models/Empleado.cs
namespace BackendPostgres.Models
{
    public class Empleado
    {
        private int idEmpleado;
        private string nombre;
        private string apellido;

        public int IdEmpleado { get => idEmpleado; set => idEmpleado = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Apellido { get => apellido; set => apellido = value; }

        public Empleado(int idEmpleado, string nombre, string apellido)
        {
            this.IdEmpleado = idEmpleado;
            this.Nombre = nombre;
            this.Apellido = apellido;
        }

        public Empleado()
        {
        }
    }
}
